inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html

```diff
@@ -1,6 +1,6 @@
 <tr class="alert {$alert['alert_status']} alert--{$alert['alert_code']}"
     id="alert_row_{$alert['id']}">
-    <td class="{$altbg} alert__avatar" width="50" align="center">
+    <td class="{$altbg} alert__avatar" width="50px" align="center">
```

Comment:
As best I can tell, the `width` attribute takes only a number or a number followed by a percentage sign, not a number followed by "px". I might be wrong about this, but if I'm not, this change should be reverted before we merge the PR.